extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesProcessor.java

```diff
@@ -403,6 +403,11 @@ private void applyBuildItems(Session session,
         configMap.put(OPENSHIFT, openshiftConfig);
         configMap.put(KNATIVE, knativeConfig);
 
+        //Replicas
+        kubernetesConfig.getReplicas()
+                .ifPresent(i -> session.resources().decorate(new KubernetesApplyReplicasDecorator(kubernetesName, i)));
+        openshiftConfig.getReplicas().ifPresent(i -> session.resources().decorate(new OpenshiftApplyReplicasDecorator(i)));
+
```

Comment:
@geoand I deleted the getReplicas() method from PlatformConfiguration and applied the decator addition for kubernetes and openshift like this. Would you have any suggestions?

Reply:
I looks good to me, :+1:

Reply:
Glad to hear @geoand  :)